### ISSUE TITLE
`initialize': wrong number of arguments

### ISSUE THREAD

**Issue by bejucoo** [wayspurrchen/moshy#7]
Hello,

I'm trying to use moshy for the first time following the instructions in the readme file but I get the following error:

```
/home/pierre/.gem/gems/slop-4.2.1/lib/slop/options.rb:26:in `initialize': wrong number of arguments (given 1, expected 0) (ArgumentError)
	from /home/pierre/.gem/gems/slop-4.2.1/lib/slop.rb:23:in `new'
	from /home/pierre/.gem/gems/slop-4.2.1/lib/slop.rb:23:in `parse'
	from /home/pierre/.gem/gems/moshy-1.0.2/bin/moshy:78:in `<top (required)>'
	from /home/pierre/.gem/bin/moshy:23:in `load'
	from /home/pierre/.gem/bin/moshy:23:in `<main>'
```

I'm not sure what's going on. I've searched the error but haven't found any answer. I have the feeling is a Ruby version issue(?). I'm new to Ruby so I can't really tell.

I have `ruby 3.0.0p0 (2020-12-25 revision 95aff21468) [x86_64-linux]` and `gem 3.2.3`

Thank you.

**Comment by Dylan-Kazak** [comment]
Same issue here, im on windows 10.
`C:/tools/ruby30/lib/ruby/gems/3.0.0/gems/slop-4.2.1/lib/slop/options.rb:26:in `initialize': wrong number of arguments (given 1, expected 0) (ArgumentError)
        from C:/tools/ruby30/lib/ruby/gems/3.0.0/gems/slop-4.2.1/lib/slop.rb:23:in `new'
        from C:/tools/ruby30/lib/ruby/gems/3.0.0/gems/slop-4.2.1/lib/slop.rb:23:in `parse'
        from C:/tools/ruby30/lib/ruby/gems/3.0.0/gems/moshy-1.0.2/bin/moshy:78:in `<top (required)>'
        from C:/tools/ruby30/bin/moshy:23:in `load'
        from C:/tools/ruby30/bin/moshy:23:in `<main>'`

**Comment by Dylan-Kazak** [comment]
> Hello,
> 
> I'm trying to use moshy for the first time following the instructions in the readme file but I get the following error:
> 
> ```
> /home/pierre/.gem/gems/slop-4.2.1/lib/slop/options.rb:26:in `initialize': wrong number of arguments (given 1, expected 0) (ArgumentError)
> 	from /home/pierre/.gem/gems/slop-4.2.1/lib/slop.rb:23:in `new'
> 	from /home/pierre/.gem/gems/slop-4.2.1/lib/slop.rb:23:in `parse'
> 	from /home/pierre/.gem/gems/moshy-1.0.2/bin/moshy:78:in `<top (required)>'
> 	from /home/pierre/.gem/bin/moshy:23:in `load'
> 	from /home/pierre/.gem/bin/moshy:23:in `<main>'
> ```
> 
> I'm not sure what's going on. I've searched the error but haven't found any answer. I have the feeling is a Ruby version issue(?). I'm new to Ruby so I can't really tell.
> 
> I have `ruby 3.0.0p0 (2020-12-25 revision 95aff21468) [x86_64-linux]` and `gem 3.2.3`
> 
> Thank you.

i figured it out, you must install an older version of ruby, i used 2.7.1

**Comment by szajbus** [comment]
> i figured it out, you must install an older version of ruby, i used 2.7.1

Or you can update to newer version of `slop`, e.g. v. 4.9.1 is compatible with ruby3

**Comment by wayspurrchen** [comment]
Hey all, y'all are correct. I just published `moshy-2.0.1` which uses `slop` `4.9.2`, ruby3, and ffmpeg5. Thanks for your patience and helpful comments!
